contracts/RelayHub.sol

```diff
@@ -296,7 +296,6 @@ contract RelayHub is IRelayHub {
         uint256 amount = stakeInfo.stake;
```

Comment:
Line 295: StakeInfo memory stakeInfo = stakes[relayManager];

copying the stakeInfo to memory seems unnecessary.  I would suggest using the "storage" keyword instead
By doing so, the next line where the stake is updated, it can be done against this storage variable